packages/tests/src.ts/test-providers.spec.ts

```diff
@@ -1096,13 +1096,17 @@ describe("Test Hedera Provider", function () {
     }).timeout(timeout * 4);
 
     it('should submit signed transaction', async function() {
```

Comment:
All the required logic is present in Testnet (autocreation + ECDSA support). Lets switch to that network otherwise the PK/Account will get its balance deleted every couple of days